Add the input output facade controller flag

SparshithNR opened the pull request:
This is to support the changes made in broccoli PR. https://github.com/broccolijs/broccoli/pull/431

stefanpenner commented:
this LGTM @thoov / @krisselden do we consider this a minor version bump?

thoov commented on index.d.ts:
```diff
@@ -133,18 +133,24 @@ export interface TransformNodeInfo extends NodeInfoCommon<"transform"> {
+
+  /**
+   * If true, Proxy to fs is enabled to provided to plugins to perform all file operations by just using
+   * plugin.input.fs.readFileSync/plugin.output.fs.writeFileSync. Defaults to false.
```
Remove .fs from this

Commit:
Add fsFacade to node-api

1. Add the input output facade controller flag
2. Add setFSFacade method to TransformNodeInfo
